sklearn/metrics/_scorer.py

```diff
@@ -410,9 +410,25 @@ def get_scorer(scoring):
     return scorer
 
 
-class _PassthroughScorer:
+class _PassthroughScorer(_MetadataRequester):
+    # Passes scoring of estimator's `score` method back to estimator if scoring
+    # is `None`.
+
     def __init__(self, estimator):
         self._estimator = estimator
+        if hasattr(estimator, "set_score_request"):
+            self.set_score_request = estimator.set_score_request
```

Comment:
So the `set_score_request` method builds some specific data for each instance which can be pickled.

But for me it seems that `estimator.set_score_request` has been an instance attribute/method before already, only belonging to estimator, instead of to the scorer instance.